cypress/e2e/participant/default-tests/playFluency-ARF.cy.js

```diff
@@ -6,22 +6,12 @@ const app = '@bdelab/roam-fluency';
 describe('Test playthrough of Fluency as a participant', () => {
   it('Fluency Playthrough Test with username/password authentication', () => {
     cy.wrap(isCurrentVersion(app)).then((isCurrentVersion) => {
-      if (isCurrentVersion) {
+      if (!isCurrentVersion) {
```

Comment:
I think this might have to come back as well as the clever CI
```suggestion
      if (isCurrentVersion) {
```

Reply:
Yep, this was just to make sure that the letter test would pass on remote. It's passed on this run, so I will uncomment this line as well as the line changing the target directory in the Gh action.